recipes/leftbar/general_row/general_row_constants.js

```diff
@@ -0,0 +1,32 @@
+export const LEFTBAR_GENERAL_ROW_TYPES = {
+  INBOX: 'inbox',
+  CONTACTS: 'contacts',
+  CHANNELS: 'channels',
+  THREADS: 'threads',
+  LOCKED_CHANNEL: 'locked channel',
+  CONTACT_CENTER: 'contact center',
+  DIALBOT: 'dialbot',
+};
+
+export const LEFTBAR_GENERAL_ROW_ICON_MAPPING = {
+  [LEFTBAR_GENERAL_ROW_TYPES.INBOX]: 'inbox',
+  [LEFTBAR_GENERAL_ROW_TYPES.CONTACTS]: 'contacts',
+  [LEFTBAR_GENERAL_ROW_TYPES.CHANNELS]: 'hash',
+  [LEFTBAR_GENERAL_ROW_TYPES.THREADS]: 'thread',
+  [LEFTBAR_GENERAL_ROW_TYPES.LOCKED_CHANNEL]: 'lock',
+  'channel bold': 'hash-bold',
```

Comment:
If there are unreads, the locked channel icon is `lock-filled`

![image](https://user-images.githubusercontent.com/1165933/220527760-7be8e31d-be54-4c73-8424-c2714cba6619.png)

![image](https://user-images.githubusercontent.com/1165933/220527378-ed4c011c-ae67-475e-9272-3bf54102c622.png)

Reply:
got it, should be there now.